debian/postinst

```diff
@@ -26,3 +26,5 @@ if [ ! -f /etc/mender/mender.conf ]; then
     # Initialize artifact_info; should be removed after MEN-2585
     echo "artifact_name=unknown" > /etc/mender/artifact_info
 fi
+
+#DEBHELPER#
```

Comment:
What's this? Special debian magic?

Reply:
Yes: special Debian magic.

It took me a bit to figure it out, but this will make the debhelper insert here all other autogenerated postinst/postrm. For our case, this meant the all the handling of the systemd service when installing, removing or upgrading.